cli/src/cli/package.rs

```diff
@@ -220,6 +241,36 @@ impl Packager {
                     meta_tree.insert(file_name.clone(), META_BIN_ID.into());
 
                     let build = Build::from_file(build_config)?;
+                    if build.steps.iter().any(|s| s.command == "cargo") {
+                        let directories = node
+                            .read_dir()?
+                            .collect::<Result<Vec<_>, _>>()
+                            .unwrap_or_default();
+
+                        directories
+                        .iter()
+                        .map(|p|p.path())
+                        .filter(|path| path.ends_with(CARGO_FILE_NAME))
+                        .for_each(|read_path|{
+
+                            File::open(read_path)
+                            .map(|mut read_file|{
+                                let mut contents = String::new();
+                                read_file
+                                .read_to_string(&mut contents)
+                                .map(|_|{
+                                    hdk_version_compare(&HDK_VERSION,&*contents)
+                                    .map(|hdk_match|{
+                                        if let false = hdk_match
+                                        {
+                                            eprintln!("There has been a version mismatch in the current zome. Current HDK Version is {}",HDK_VERSION.to_string())
```

Comment:
Here's a wording suggestion for the error, and also we should include the file name.  If possible, can we get the version found out of the contents to print that too?  i.e. hdk_version_compare could return the value found so we can print it in the warning?  Something like:
```suggestion
                                            eprintln!("WARNING: The HDK version found in {:?} does not match the current version.\n If you are seeing compilation problems, update the version in your Cargo.toml files to the current version: {}", read_path, HDK_VERSION.to_string())
```

Reply:
Absolutely, will implement that right now